eatin-server/migrate/V6__create_table_user_recipes.pgsql

```diff
@@ -2,13 +2,13 @@ create table if not exists userRecipes
 (
```

Comment:
have you deleted the relevant row in the flyway table?

Reply:
what do you mean?

Reply:
when a script is added, a row in the `flyway_schema_history` table is added for that script.
each row contains data about the script, and the file checksum. the checksum is saved in order the validate that the checksum of each script remains the same - meaning, the script has not changed. this validation runs every time we run the `npm run migrate` and in the future every time we will run CI\CD.
so there are two options - 
1. delete the row of the script in the table. the flyway will recognize a new script that is not saved, so it will run it and save it.
2. create a new script, that alters the constraints. this is the better option, but because we are not in production do the first option.

<img width="996" alt="image" src="https://user-images.githubusercontent.com/37041432/230606381-a46296f1-c687-439c-9350-b768ffdeb4de.png">